test/service/test_pypi.py

```diff
@@ -22,8 +22,10 @@ def get(self, url, **kwargs):
 
 def test_pypi(cache_dir):
     pypi = service.PyPIService(cache_dir)
-    dep = service.Dependency("jinja2", Version("2.4.1"))
-    results: List[service.VulnerabilityResult] = dict(pypi.query_all([dep]))
+    dep = service.ResolvedDependency("jinja2", Version("2.4.1"))
+    results: Dict[service.Dependency, List[service.VulnerabilityResult]] = dict(
```

Comment:
This isn't related to the change, I just noticed the type hints in this file are wrong. Is there a reason we don't run MyPy over the test code, or should I follow up with a change for that?

Reply:
No good reason -- I was just lazy about only applying `mypy` to the library source. Yeah, let's do a follow-up PR for linting the tests as well.